### ISSUE TITLE
Is This Project Still Active?

### ISSUE THREAD

**Issue by pranaypaine** [Semantic-Org/Semantic-UI#7078]
✖ USAGE QUESTIONS → Use these dedicated resources:
      Docs - http://www.semantic-ui.com
      Chat - https://gitter.im/Semantic-Org/Semantic-UI
      SO - https://stackoverflow.com/questions/tagged/semantic-ui?sort=votes

✔ Enhancements → Be specific. Assume backwards compatibility is a necessity. Suggest implementation when possible.
✔ BUGS → ❤❤❤. Keep in mind some bugs may not be immediately fixable due to backwards compatibility or CSS limitations.

### Title (Put in field above)
Use the format: [Component] Component Should Do X
i.e. [Checkbox] onChange Should Fire When Update Triggered via DOM

### Version
x.y.z

### Testcase
[Fork, update, and replace to show the bug]:
https://jsfiddle.net/ca0rovs3/

**Comment by KonradHoeffner** [comment]
In this case I suggest archiving this repository, so that it is clear to others.

**Comment by prudho** [comment]
Yup, this project was rather dead until... let's see... yesterday !

And now the actual benevolent dictator of this project, which is the only one that can make a release, archive the repository... is back, AFTER A 4 YEAR HIATUS !!! 🥳

Of course he didn't talk about the Fomantic-UI fork, or the fact that a lot of willing people tried to contact him to help, maintain, improve SUI...

Oh, and he is marking all @lubber-de comments as spam, and it seem he even banished him from this repository !!! Sure ! People, should not know that such a fork exists, with an active team that improved almost every part of this framework, and is trying his best to respect the project's legacy.

Yeah... i'm really pissed off right now. Feel free to ban me too, that's the only way you know to communicate with the people that tried to help you @jlukic.
